### ISSUE TITLE
frequent database dump and license

### ISSUE THREAD

**Issue by tcecyk** [jonathanklee/Sapio#17]
would you offer the database with a permissive license as frequent data dump? looking at other compat-databases:

* protondb offers their dataset as ODbL at https://github.com/bdefore/protondb-data
* techlore Plexus App did dump into a .csv, then .json in it's repository, but since moving to an API those dumps seem dated and I can't find an explicit license on the data

**Comment by jonathanklee** [comment]
@tcecyk If people are interested in a database dump, sure, why not. What kind of application do you envision for this database dump?

**Comment by tcecyk** [comment]
two things come to mind:
* it's mostly assurance for data contributors for a chance at continuation if a project stops or the (api) host is offline. Kind of an appreciation of their contribution
* offering a dump will make people play with the data, integrate it in their own query services. Think of what /e/ AppLounge does with appid tracker scoring from ETIP: you could display some estimate on the compatibility at the same time. I think you might have had something like that in mind when starting it

**Comment by jonathanklee** [comment]
@tcecyk Yes I had those things in mind. Regarding third party services that would like to obtain the data, I would rather think of documenting the API properly so that people can use data in a more dynamic way,
